packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -375,18 +405,21 @@ function get_pattern(segments, add_trailing_slash) {
  * @param {string} dir
  * @param {string} path
  * @param {import('types/internal').Asset[]} files
+ * @param {string[]} excludedPaths Paths relative to dir which should be excluded from files list.
  */
-function list_files(dir, path, files = []) {
+function list_files(dir, path, files = [], excludedPaths = []) {
```

Comment:
SvelteKit uses underscore for internal variables. camelCase is used only for external APIs
```suggestion
function list_files(dir, path, files = [], excluded_paths = []) {
```

Reply:
Thanks. Made the changes. 
Any place I can document this or it's already documented at.